### PR TITLE
raidboss: cn 5.01 eden normal

### PR DESCRIPTION
**Pull request by Bluefissure**
update Eden normal timeline/triggers of cn patch 5.01.

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e1s.js [hunk 1]
```diff
@@ -600,16 +600,53 @@
...
+        'Eden Prime': '',
+        'Arcane Sphere': '',
       },
       'replaceText': {
         'attack': '攻击',
-        'Unknown Ability': 'Unknown Ability',
+        'Vice of Vanity': '',
+        'Vice of Thievery': '',
+        'Vice of Sloth': '',
+        'Vice of Pride': '',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e1s.js]: Did you mean to leave all of these blank?

Comment (Bluefissure) [reply ui/raidboss/data/05-shb/raid/e1s.js]: I was trying to fill these but it seems these translations are not released in current exd/csv files of CN client, so I just leave them blank and will finish them when savage is released.

Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e1s.js]: Ok! I just figured they'd all mostly be the same as e1n, but waiting is fine!

Comment (Bluefissure) [reply ui/raidboss/data/05-shb/raid/e1s.js]: Yeah mostly, but not all of them, I need to update them eventually so I’m going to wait till then.

**Pull request merged**